src/ingredients/dto/update.ingredient.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+
+export class UpdateIngredientDto {
+  @ApiProperty({
+    type: String,
+    description: 'Ingredient title',
+    required: false,
+  })
+  @IsString()
+  @IsNotEmpty()
+  @IsOptional()
+  title?: string;
+
+  @ApiProperty({
+    type: Boolean,
+    description: 'Ingredient status',
+    required: false,
+  })
+  @IsOptional()
+  available: boolean;
```

Comment:
Make sure to append `?` to this field, since it's an optional

Reply:
Okkk